compare-optimizer.sh

```diff
@@ -13,9 +13,13 @@ fi
 
 set -e -x
 
-# run develop version of cmdstan with the nightly stanc3 binary
-cd cmdstan; make clean-all; 
-make -j4 build; make -j4 examples/bernoulli/bernoulli; ./bin/stanc --version; cd ..
+# run with the default optimization level
+cd cmdstan; make clean-all; make -j4 build;
+if [ -n "$3" ] ; then
+    rm cmdstan/bin/stanc
+    cp "$3" cmdstan/bin/stanc
```

Comment:
I believe the current directory when this is run is `cmdstan` already, no?

Reply:
Indeed, fixed.